spec/system/hiring_staff_have_to_accept_terms_spec.rb

```diff
@@ -2,13 +2,12 @@
 
 RSpec.describe "Hiring staff accepts terms and conditions" do
   let(:school) { create(:school) }
-  let(:oid) { "a-valid-oid" }
-  let(:current_publisher) { Publisher.find_by(oid: oid) }
+  let(:current_publisher) { Publisher.first }
```

Comment:
Where's the first publisher coming from?

Reply:
E.g. line 10.